src/docs/wtstats.dox

```diff
@@ -0,0 +1,47 @@
+/*! @page wtstats Visualizing performance with wtstats
+
+The WiredTiger distribution includes the \b wtstats tool that can be used to
+examine information generated using statistics logging (see @ref
+statistics_log).
+
+After running an application with statistics logging configured, the
+statistics output files will be in the database home directory.  By default,
+these are named \c WiredTigerStat.* .  In the database home directory, run
+this command, replacing \c ... with the path to the WiredTiger installation
+directory:
+\code{.sh}
+python .../tools/wtstats.py WiredTigerStat.*
+\endcode
+
```

Comment:
I have two minor comments:
1.  "Another way to collect all the stats files" -> "Another way to process all the stats files..." 
2.  In the three locations, I suggest replacing "..." with "<pathname>".  To me three dots looks too much like two dots.
Otherwise this all looks good.